_includes/configuration/toolbar-mode.md

```diff
@@ -6,6 +6,25 @@
 
 The `toolbar_mode` option is used to extend the toolbar to accommodate the overflowing toolbar buttons. This option is useful for small screens or small editor frames.
 
+When the toolbar_mode is set to `'floating'` or `sliding`, the editor will move toolbar buttons to the toolbar drawer based on:
+
+- The defined toolbar groups.
+- The width of the editor.
+
+For example. If the following toolbar configuration is wider than the editor, the items before the separator (`|`) will appear in the main toolbar and the rest will be moved to the toolbar drawer.
+
+```js
+tinymce.init({
+    selector: 'textarea',
+    toolbar: 'undo redo styleselect bold italic alignleft aligncenter alignright alignjustify | bullist numlist outdent indent'
+});
+```
+
+Two situations to consider when organising the toolbar:
+
+- When there is two or more toolbar button groups, the main toolbar will show as many complete, sequential toolbar groups as possible within the width of the editor.
```

Comment:
is/are?

Reply:
Should we add "Any remaining toolbar button groups will be moved to the toolbar drawer." or something to be explicit?

Reply:
Thanks! fixed: https://github.com/tinymce/tinymce-docs/pull/1612/commits/220ff653fe1673dd4354a87ea688eec2fe464b37